cookbook/specs/near.json

```diff
@@ -90,8 +90,9 @@
                                 "extra_compute_units": 0,
                                 "parsers": [
                                     {
-                                        "parse_path": ".finality",
-                                        "parse_type": "BLOCK_LATEST"
+                                        "parse_path": ".params.finality",
```

Comment:
we don't want to match every finality here, we need some way to only match if it's finality: final, let's add a rule field to the parser, not just a value